content/en/mobile/_index.md

```diff
@@ -426,6 +415,29 @@ On the Traces page, you can view and search all of the traces that you have acce
 
 For example, to filter on traces with the tag `#env:prod` or the tag `#test`, use the query `"env:prod" OR test`. Click into individual services to expand associated spans, and select spans to view info, errors, and related logs. You can also open traces from services and logs.
 
+## Notifications
+
+The mobile app is able to send out regular as well as critical notifications. As a reminder, critical notifications can circumvent your Do Not Disturb (DnD) mode. This is however only possible if you **explicitly** grant us permission to do so on your device. Critical notifications are only supported in the context of [Datadog On-Call][24].
+
+### Push Notifications for On-Call
+To properly set up your mobile device for On-Call, please follow [this guide][25]. Once set up, make sure you visit your [On-Call profile][26] and select your mobile device in your notification preferences.
+
+### Push Notifications for Incidents
+To stay up to date on new and ongoing incidents, Incident Management allows you to receive push notifications via [Incident Notification Rules][13]. Here's a step-by-step guide on how to set that up:
+
+1. Navigate to **Account**.
+2. Click **Notifications**.
+3. Select the toggle for **Enable Notifications**.
```

Comment:
Where are the first 3 steps happening? Is this referring to the mobile device settings, the Datadog app, or the Datadog Mobile App? If it's the Datadog app, a link to the page would be good. If it's a mobile device, are the instructions the same for Android and iOS?

Reply:
hey, yep this is for the datadog mobile app and they are indeed the same for android and iOS